wp-content/themes/core/assets/css/src/theme/actions/actions.pcss

```diff
@@ -19,7 +19,9 @@
  * ----------------------------------------------------------------------------- */
 
 .a-btn,
-a.a-btn {
+a.a-btn,
```

Comment:
Here starts examples of where we need need to be able to apply styles to our "kitchen sink" as well as WordPress core blocks, thus needing both selectors in the block editor stylesheet.  Most of the `t-sink` and `s-sink` selectors are now duped (rather than replaced) to accommodate this.

Reply:
ya we'll have to resolve, I'll get thinking on this tomorrow, merge for now :)